apps/address-service/domains/common/utils/services/InjectionsSeeker.spec.js

```diff
@@ -0,0 +1,11 @@
+const { InjectionsSeeker } = require('@address-service/domains/common/utils/services/InjectionsSeeker')
+
+describe('Injections seeker', () => {
+    test('Search parts extracted correctly', async () => {
+        const injectionsSeeker = new InjectionsSeeker('Some _- +== string with a comma: , ...  Hello world!')
```

Comment:
Why the class is named `InjectionsSeeker`, if it just extracts words from provided string?

Reply:
`InjectionsSeeker` searches for injections to inject them into the suggestions array.
This test ensures the search words are extracted correctly from the search string.
There are some tests for correct injecting (see AddressInjection.test.js)

Reply:
may we you just need a function?

Reply:
Not a single function but several ones. I've decided to encapsulate them in class. It is convenient for testing too.
But yes, it's also possible to collect these functions within some module.

Reply:
Classes nightmare is in many places of the project ;)
If a class instance lives in scope of a short function call, — it's not a use-case the class was designed for…

Reply:
Some classes were refactored to functions. But data providers weren't, because we use an abstraction within code.